core/util/src/main/scala/net/liftweb/util/Props.scala

```diff
@@ -328,5 +378,18 @@ object Props extends Logger {
         Map()
     }
   }
+
+  private var prepended:List[java.util.Map[Object, Object]] = Nil
+  private var appended:List[java.util.Map[Object, Object]] = Nil
+  private var interpolators:List[java.util.Map[Object, Object]] = Nil
```

Comment:
I expect `System.env` and `System.properties` to be used here, and both of those are mutable maps.  If we go with Scala maps, then we will take an snapshot at the time it is passed to us.  While I hope that not many people are mutating their system properties, I don't feel good about having it passed to us then we no longer call through to the map.